.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Remove this, please. You can setup global Git config on your machine to ignore files for your editor.